ts/features/bonus/cgn/components/detail/eyca/EycaDetailComponent.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from "react";
+import { connect } from "react-redux";
+import { fromNullable } from "fp-ts/lib/Option";
+import { useEffect } from "react";
+import { ActivityIndicator } from "react-native";
+import { GlobalState } from "../../../../../../store/reducers/types";
+import { Dispatch } from "../../../../../../store/actions/types";
+import {
+  eycaInformationSelector,
+  isEycaDetailsLoading
+} from "../../../store/reducers/eyca/details";
+import {
+  cgnEycaActivation,
+  cgnEycaActivationStatusRequest
+} from "../../../store/actions/eyca/activation";
+import { EycaCard } from "../../../../../../../definitions/cgn/EycaCard";
+import {
+  cgnEycaActivationLoading,
+  cgnEycaActivationStatus
+} from "../../../store/reducers/eyca/activation";
+import { CardPending } from "../../../../../../../definitions/cgn/CardPending";
+import EycaStatusDetailsComponent from "./EycaStatusDetailsComponent";
+import EycaPendingComponent from "./EycaPendingComponent";
+import EycaErrorComponent from "./EycaErrorComponent";
+
+type Props = ReturnType<typeof mapStateToProps> &
+  ReturnType<typeof mapDispatchToProps>;
+
+const EycaDetailComponent: React.FunctionComponent<Props> = (props: Props) => {
```

Comment:
```suggestion
const EycaDetailComponent = (props: Props) => {
```
Could you also leave a description of this component?

Reply:
Fixed in 059777b